fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalJdbcScan.java

```diff
@@ -30,28 +31,34 @@
 
 import java.util.List;
 import java.util.Optional;
+import java.util.Set;
 
 /**
  * Physical jdbc scan for external catalog.
  */
 public class PhysicalJdbcScan extends PhysicalCatalogRelation {
 
+    private final Set<Expression> conjuncts;
+
     /**
      * Constructor for PhysicalJdbcScan.
      */
     public PhysicalJdbcScan(RelationId id, TableIf table, List<String> qualifier,
-            Optional<GroupExpression> groupExpression, LogicalProperties logicalProperties) {
+            Optional<GroupExpression> groupExpression, LogicalProperties logicalProperties, Set<Expression> conjuncts) {
         super(id, PlanType.PHYSICAL_JDBC_SCAN, table, qualifier, groupExpression, logicalProperties);
+        this.conjuncts = conjuncts;
     }
 
     /**
      * Constructor for PhysicalJdbcScan.
      */
     public PhysicalJdbcScan(RelationId id, TableIf table, List<String> qualifier,
             Optional<GroupExpression> groupExpression,
-            LogicalProperties logicalProperties, PhysicalProperties physicalProperties, Statistics statistics) {
+            LogicalProperties logicalProperties, PhysicalProperties physicalProperties, Statistics statistics,
+            Set<Expression> conjuncts) {
         super(id, PlanType.PHYSICAL_JDBC_SCAN, table, qualifier, groupExpression,
                 logicalProperties, physicalProperties, statistics);
+        this.conjuncts = conjuncts;
```

Comment:
this.conjuncts = ImmutableSet.copyOf(Objects.requireNonNull(conjuncts, "conjuncts should not be null"));